Bio/codonalign/codonseq.py

```diff
@@ -210,7 +210,7 @@ def translate(
                 )
         return "".join(amino_acids)
 
-    def toSeq(self, alphabet=generic_dna):
```

Comment:
the only usages of this method in Biopython are two instances of:

```
        alignments = [SeqRecord(rec.seq.toSeq(), id=rec.id) for rec in self._records]
```

However this method is not private and maybe breaks users functionality? unsure.

Reply:
We're dropping alphabet usage as part of the next release. While on very commonly used function signatures (like ``Bio.SeqIO.parse``) I think we need to consider something to ease the transition, here though removal seems OK. @mdehoon do you agree?

Reply:
Yes let's remove it.